perfdash/github-configs-fetcher.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type githubDirContent struct {
+	Name        string `yaml:"name"`
+	Path        string `yaml:"path"`
+	DownloadUrl string `yaml:"download_url"`
+	Type        string `yaml:"type"`
+	Url         string `yaml:"url"`
+}
+
+/*
+Gets config paths from github directory. It uses github API, which is documented
+here: https://developer.github.com/v3/repos/contents/
+
+Example url: https://api.github.com/repos/pierewoj/test-infra/contents/config/jobs/kubernetes/sig-release/release-branch-jobs
+
+Different branch can be specified by appending "?ref=branch-name" parameter at the end
+of the url.
+*/
+func GetConfigsFromGithub(url string) ([]string, error) {
```

Comment:
Comment for this method needs to start with `GetConfigsFromGithub ...`
golint complains about that

Reply:
done

Reply:
We generally try to not have comments in /* .. */

Can you please change to simply start each line with //

Reply:
done